web/ui/mantine-ui/src/pages/query/DataTable.tsx

```diff
@@ -203,14 +205,7 @@ const DataTable: FC<DataTableProps> = ({ expr, evalTime, retriggerIdx }) => {
 };
 
 const histogramTable = (h: Histogram): ReactNode => (
-  <Table>
-    <Table.Thead>
-      <Table.Tr>
-        <Table.Th style={{ textAlign: "center" }} colSpan={2}>
-          Bucket counts
-        </Table.Th>
-      </Table.Tr>
-    </Table.Thead>
+  <Table withTableBorder fz="xs">
```

Comment:
what is `fz`? Can't seem to find anything in the docs

Reply:
Font size :) See https://mantine.dev/styles/style-props/ - yeah, looks like the site search on mantine.dev doesn't index minute details like that.

Reply:
Ahh nice! 
Yeah this makes sense